packages/sit-onyx/src/components/OnyxInput/types.ts

```diff
@@ -75,10 +75,16 @@ export type OnyxInputProps = DensityProp &
     loading?: boolean;
     /**
      * Maximum number of characters that are allowed to be entered.
+     * Warning: when the value is (pre)set by code,
+     * the input invalidity will not be detected by the browser, it will only show as invalid
+     * as soon as a user interacts with the input (types something).
```

Comment:
```suggestion
     * Warning: when the value is (pre)set programatically,
     * the input invalidity will not be detected by the browser, it will only turn invalid
     * as soon as a user interacts with the input (types something).
```